packages/manager/apps/dedicated/client/app/index.js

```diff
@@ -6,14 +6,19 @@ import {
 } from '@ovh-ux/manager-preloader';
 import { bootstrapApplication } from '@ovh-ux/manager-core';
 import { Environment } from '@ovh-ux/manager-config';
+import { User } from '@ovh-ux/manager-models';
 
-attachPreloader(Environment.getUserLanguage());
-
-bootstrapApplication('dedicated').then(({ region, message }) => {
+bootstrapApplication('dedicated').then(({ message, region, user }) => {
   if (message) {
     displayMessage(message, Environment.getUserLanguage());
   }
-
+  const currentUser = new User(user);
+  attachPreloader(
```

Comment:
Here you attach the preloader after receiving the `/configuration` response, the principle of the preloader is to be displayed as soon as possible.
If the HTTP call take a lot of time, nothing is displayed during loading.

Reply:
This change has been removed, after confirmation with the PU. This is not needed.